tests/test_geo_utils.py

```diff
@@ -5,58 +5,70 @@
 
 import pytest
 
-from sentinelhub import CRS, BBox, geo_utils
+from sentinelhub import CRS, BBox
+from sentinelhub.geo_utils import (
+    bbox_to_dimensions,
+    bbox_to_resolution,
+    get_image_dimension,
+    get_utm_crs,
+    pixel_to_utm,
+    to_wgs84,
+    transform_point,
+    utm_to_pixel,
+    wgs84_to_pixel,
+    wgs84_to_utm,
+)
 
+BBOX_WGS84 = BBox(((111.644, 8.655), (111.7, 8.688)), CRS.WGS84)
+BBOX_UTM = BBox(((570851.8316965176, 956770.2133183606), (577006.4223865959, 960429.6742984429)), CRS("32649"))
+BBOX_POP_WEB = BBox(((12428153.230124235, 967155.4076988235), (12434387.12160866, 970871.4287682716)), CRS.POP_WEB)
 
-def test_wgs84_to_utm() -> None:
-    x, y = geo_utils.wgs84_to_utm(15.525078, 44.1440478, CRS.UTM_33N)
-    assert (x, y) == pytest.approx((541995.694062, 4888006.132887), rel=1e-8)
 
+@pytest.mark.parametrize("input_bbox, expected_bbox", [(BBOX_WGS84, BBOX_UTM), (BBOX_WGS84, BBOX_POP_WEB)])
```

Comment:
hold on, the function is named `wgs84_to_utm`, why does it work with `POP_WEB`?

Reply:
function have wrong name, test updated